### PR TITLE
Feature/inactive projects view dev

### PR DESCRIPTION
**Pull request by DirkSuelmann**

### CODE REVIEW

**Review by LiWeiYeh — APPROVED** [review]
Congrats on your first PR! It looks good! It seems like the editing functionality has not been implemented yet in this PR though. I approve this PR, but be sure to handle edit functionality before you set your work item on "done".

**Review by LiWeiYeh — CHANGES_REQUESTED** [review]
Whoops, my previous PR went through so there are some merge conflics. Resolve them first!

File: src/app/project-administration/inactive-projects/inactive-projects.interface.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export interface InactiveProjectsInterface extends ObjectBase {
```
Comment (PatrickR079) [inline src/app/project-administration/inactive-projects/inactive-projects.interface.ts:1]: You can use the 'active-projects' interface for this. My request when you apply this change: can you rename the 'active-projects' interface to 'projects' interface?

**Review by PatrickR079 — CHANGES_REQUESTED** [review]

File: src/app/project-administration/inactive-projects/inactive-projects.interface.ts (hunk 1 above)
Comment (DirkSuelmann) [reply src/app/project-administration/inactive-projects/inactive-projects.interface.ts]: You would think that the data would be the same, but it is slightly different. So i've added a ``ProjectsInterface`` as a base, and made the ``ActiveProjectsInterface`` extend this interface.

File: src/app/project-administration/backend.mock.service.ts [hunk 2]
```diff
@@ -1,19 +1,25 @@
 import { Injectable } from '@angular/core';
 import { Observable, of } from 'rxjs';
 import { IBackendService } from './backend.service.interface';
-import { ActiveProjectsInterface } from './active-projects/active-projects.interface';
+import { ProjectsInterface } from './projects.interface';
```
Comment (LiWeiYeh) [inline src/app/project-administration/backend.mock.service.ts:4]: Should be InactiveProjectsInterface

File: src/app/project-administration/backend.mock.service.ts [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 import { ProjectInterface } from './project/project.interface';
+import { ActiveProjectsInterface } from './active-projects/active-projects.interface';
 
 @Injectable()
 export class BackendMockService implements IBackendService {
   getActiveProjects(): Observable<ActiveProjectsInterface[]> {
     return of(ACTIVE_PROJECTS_TEST_DATA);
   }
 
+  getInactiveProjects(): Observable<ProjectsInterface[]> {
```
Comment (LiWeiYeh) [inline src/app/project-administration/backend.mock.service.ts:19]: Should be of type InactiveProjectsInterface

File: src/app/project-administration/backend.service.interface.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Observable } from 'rxjs';
+import { ProjectsInterface } from './projects.interface';
```
Comment (LiWeiYeh) [inline src/app/project-administration/backend.service.interface.ts:2]: InactiveProjectsInterface

File: src/app/project-administration/backend.service.ts [hunk 5]
```diff
@@ -1,12 +1,13 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
+import { ProjectsInterface } from './projects.interface';
```
Comment (LiWeiYeh) [inline src/app/project-administration/backend.service.ts:4]: InactiveProjectsInterface

File: src/app/project-administration/backend.service.ts [hunk 6]
```diff
@@ -1,12 +1,13 @@
...
 import { ActiveProjectsInterface } from './active-projects/active-projects.interface';
+import { ProjectInterface } from './project/project.interface';
```
Comment (LiWeiYeh) [inline src/app/project-administration/backend.service.ts:6]: InactiveProjectsInterface

File: src/app/project-administration/backend.service.ts [hunk 7]
```diff
@@ -16,6 +17,10 @@ export class BackendService implements IBackendService {
     return this.http.get<ActiveProjectsInterface[]>('resource/SESSION/1/Active_32_projects');
   }
 
+  public getInactiveProjects(): Observable<ProjectsInterface[]> {
```
Comment (LiWeiYeh) [inline src/app/project-administration/backend.service.ts:20]: InactiveProjectsInterface

**Review by LiWeiYeh — CHANGES_REQUESTED** [review]
Apply the change you made in your last commit, which is changing the ProjectInterface into InactiveProjecstInterface and ActiveProjectsInterface, The change of InactiveProjectsInterface has not been applied to other files, e.g. backend.services

File: src/app/project-administration/backend.service.ts (hunk 6 above)
Comment (LiWeiYeh) [reply src/app/project-administration/backend.service.ts:5]: Is supposed to be for the Project

**Review by LiWeiYeh — APPROVED** [review]

**Review by PatrickR079 — APPROVED** [review]

**Pull request merged**